sonarqube/httpHelpers.go

```diff
@@ -26,17 +29,17 @@ type Paging struct {
 }
 
 // helper function to make api request to sonarqube
-func httpRequestHelper(client *retryablehttp.Client, method string, sonarqubeURL string, expectedResponseCode int, errormsg string) (http.Response, error) {
+func httpRequestHelper(client *retryablehttp.Client, method string, sonarqubeURL url.URL, expectedResponseCode int, errormsg string) (http.Response, error) {
```

Comment:
I'm not objecting to the change but why make this a url now and then have to `.String()` it everywhere?